test/bin/create_local_repo.sh

```diff
@@ -40,3 +74,5 @@ make_repo "${LOCAL_REPO}" "${RPM_SOURCE}"
 make_repo "${NEXT_REPO}" "${NEXT_RPM_SOURCE}"
 make_repo "${YPLUS2_REPO}" "${YPLUS2_RPM_SOURCE}"
 make_repo "${BASE_REPO}" "${BASE_RPM_SOURCE}"
+
+download_repo "${YMINUS1_REPO}" "4.$(previous_minor_version)"
```

Comment:
What's this for?

Reply:
To re-implement `ci/prow/microshift-rpm-upgrade` job, we need Y-1 microshift RPMs. 
Since we have no rshm at the moment, we cannot pull the RPMs directly from the repo and install on the host.
Hence, we copy the RPMs to a local repo alongside others that we build.

Such repos can then be installed on a VM assuming that all the dependencies are furnished beforehand.